tasks/dnsmasq.yml

```diff
@@ -37,7 +37,7 @@
     mode: 0644
   become: true
   notify: restart dnsmasq
-  when: "{{ item.when }}"
+  when: item.when
```

Comment:
```suggestion
  when: "{{ dnsmasq_item.when }}"
```

See #488.

Reply:
@nre-ableton 
The lint error was about jinja expressions being used in `when`. It only expected pure python expressions. Have you tried this out by lint?

Reply:
Duplicate